plugins/bundle/plugin.go

```diff
@@ -598,6 +599,17 @@ func (p *Plugin) activate(ctx context.Context, name string, b *bundle.Bundle) er
 			Bundles:  map[string]*bundle.Bundle{name: b},
 		}
 
+		if p.manager.Info != nil {
+			x := p.manager.Info.Get(ast.StringTerm("authorization_scheme_basic"))
```

Comment:
The reason for this is the authz schemes OPA supports in kept in the `server` package and we need this info in the bundle plugin. So to avoid having to import the server package in the bundle plugin (this would result in an import error) or having to duplicate/move the authz scheme constants somewhere, I thought we can go with this. I originally had this as a string term `authorization_scheme` but then went with the boolean route.

Reply:
We can change the name to something like `authorization_enabled` to give an indication whether or not authz is enabled. WDYT?

Reply:
Yes, I like that 👍 .